MathlibTest/TermReduce.lean

```diff
@@ -73,7 +73,7 @@ instance c : C :=
   {}
 
 set_option pp.all true in
-/-- info: @C.mk (@B.toA (@B.mk A.mk)) : C -/
+/-- info: @C.mk A.mk : C -/
```

Comment:
I'm assuming that this change in a test is expected and desired?

Reply:
Yes, the term it produces is better now. `B.toA` is a left inverse of `B.mk`.